scene/3d/SCsub

```diff
@@ -4,6 +4,5 @@ Import("env")
 
 if env["disable_3d"]:
     env.add_source_files(env.scene_sources, "node_3d.cpp")
```

Comment:
Is this still needed too? You're no longer registering it.

Reply:
Yes, it's still needed, Viewport and PackedScene still depend on it (for now).

Reply:
Do they still work even without `ClassDB::register_class<Node3D>();`?

Reply:
Yes, I tested that an export template with 3D disabled is able to load a 2D-only project and a UI-only project (therefore Viewport and PackedScene must be working).